release-notes/v2.4.0.md

```diff
@@ -0,0 +1,48 @@
+# Agent 2.4.0 release notes 
+
+Agent release 2.4.0 is published! This release requires a small amount of action from you.
+
+## New deployment files
+We have slightly adjusted the artifact files
+
+run 
+
+`rm -rf /data/dist/`
+
+
+You can download the new files with
+
+```
+curl -sSfL https://registry.npmjs.org/@beamer-bridge/deployments/-/deployments-2.0.2.tgz |
+     tar xz -C data --strip-components=1 package/dist
+```
+
+Please run this command inside of the ROOT folder of your `run-your-own-agent` project.
+
+You will see now a new file structure like
+
+```
+./data/dist/artifacts/mainnet
+./data/dist/abis/mainnet
+```
+
+The `artifacts` directory contains all the information about the deployed contracts.
+The `abis` directory contains the necessary ABIs of the contracts.
+
+
+## Adding Polygon zkEVM
+With this version we are supporting Polygon zkEVM as a new rollup.
+
+* Add Polygon as a chain to you agent.conf as in https://github.com/beamer-bridge/run-your-own-agent/blob/ac5180c70ae6983e77e5bf3880bc349c8fe19959/data/agent-mainnet.conf#L28
+* Add the token addresses for Polygon (lines with chain id `1101`) to your tokens like in https://github.com/beamer-bridge/run-your-own-agent/blob/main/data/agent-mainnet.conf#L43-L66
+* fund your account on Polygon
+
+## Update as always
+
+* change the image in your docker-compose.yml to v.2.4.0
+* run 
+
+```sh
+docker compose down
+docker compose up -d
```

Comment:
we should especially only run agent imo, i have some worries regarding health check part